### ISSUE TITLE
Follicular bronchiolitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6254]
**Preferred term label:**
Follicular bronchiolitis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 24116927  1/1
https://omim.org/entry/606367
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Follicular bronchiolitis (FB) is a polyclonal hyperplasia of bronchiolar associated lymphoid tissue (BALT) characterized by the is characterized by the development of lymphoid follicles with germinal centers in walls of the small airways. PMID:32047693

Comment: Follicular bronchiolitis is a feature of lymphoproliferative pulmonary diseases (LPDs) and is associated with connective tissue diseases (rheumatoid arthritis, Sjoegren's syndrome), interstitial lung diseases, disorders of mixed collagen-vascular, obstructive diseases of airways and states immunodeficiency including HIV and common variable immunodeficiency.

**Comment by mgriese** [comment]
Oh , I just saw, that we are already beyond this point of the discussion, I just raised with lymphcytic infiltration in the lungs.

**Comment by mgriese** [comment]
The problem now is, is this term a child of the term lymphocytic infiltration or are they siblings?

**Comment by mgriese** [comment]
One could add a hint that quatitation is a way to assess the presence and extent of FB (or LIP) (e.g. as we detailed in 32040879, based on an older paper not accessible 4586901) .

**Comment by pnrobinson** [comment]
@mgriese  Should we change the comment to this based on your paper?

Lymphocytic interstitial pneumonia (LIP) and follicular bronchiolitis (FB) are assumed to be pathological immunologic responses to different triggers and are particularly associated with autoimmune and infectious diseases as well as immunodeficiency. LIP can be defined as a dense, predominantly lymphocytic interstitial infiltrate that expands alveolar septa. In contrast, FB can be defined as a pattern of numerous lymphoid follicles associated with bronchioles and the immediate peribronchiolar interstitium. A grading system can be use to assess presence and extent of LIP and FB. Lymphocyte aggregates are defined as the aggregation of 50 or more lymphocytes per high-power field. Tissue was scored as grade 0 in the absence of lymphocyte aggregates, grade 1 with 0 to 1 aggregate/10 mm 2 , grade 2 with greater than 1 aggregate/10 mm 2 , grade 3 with greater than 1 aggregate/10 mm 2 and the presence of focal lymphoid interstitial pneumonia, and grade 4 with greater than 1 aggregate/10 mm 2 and the presence of diffuse LIP. 
PMID:32040879

**Comment by pnrobinson** [comment]
addded
